app/components/chart-preview.tsx

```diff
@@ -568,6 +568,7 @@ const ChartPreviewInner = ({
                   chartConfig={chartConfig}
                   dashboardFilters={state.dashboardFilters}
                   components={allComponents}
+                  showVisualizeLink
```

Comment:
@noahonyejese I think we should only show this when a chart is published, the current behavior here is correct; see how it looks in the design, e.g. here: https://www.figma.com/design/Hn9wvuEYLUmdJtg4KwZYUx/visualize.admin.ch---design?node-id=2805-554265&m=dev